src/core/tracker.cpp

```diff
@@ -390,6 +394,11 @@ void Tracker::replayPositionInformationList( const QList<GnssPositionInformation
   {
     start();
   }
+
+  const qint64 endTime = QDateTime::currentMSecsSinceEpoch();
+  qDebug() << "---";
+  qDebug() << ( endTime - startTime ) << "ms";
+  qDebug() << "---";
```

Comment:
I guess we want to get rid of these again

Reply:
@m-kuhn , hmm, wait, I think I might have messed up my rebase, this was supposed to be a cleaned up qInfo() to help us identify further bottlenecks in the future.

Reply:
@m-kuhn , OK, rebase fixed, sorry for the noise.